README.md

```diff
@@ -2,4 +2,149 @@
 
 **This is not an official Google product.**
 
-Privacy data mapping and related plans management.
+## Background
+
+Privacy data management is the process of collecting, storing, using, and
+disposing of personal data in a way that protects the privacy of individuals. It
+is a critical part of any organization that collects or uses personal data, and
+organizations are typically required to maintain compliance with policies set by
+regulatory bodies.
+
+To ensure that organizations maintain compliance with policies set by regulatory
+bodies, people need to know the following: 
+* The requirements for what teams
+must do, driven by legal requirements or external commitments (aka Privacy
+Policy). This includes translating the comprehensive external legal requirements
+into requirements that are tailored to products and services of the
+organizations. 
+* Where the user data is stored. This includes understanding the
+different systems and databases that store personal data, as well as the
+physical locations where user data is stored.
+* Which privacy policy applies to
+the datastore that stores user data(aka Data Mapping). This includes
+understanding how the organization's privacy policies are applied to different
+systems and databases. 
+* The visibility of the privacy compliance. This includes
+being able to track and monitor the organization's compliance with its privacy
+policies and applicable laws and regulations.
+
+PMAP provides a solution for the first three problems. We are working on a
+solution to provide visibility of privacy compliance in the near future.
```

Comment:
```suggestion
To ensure that organizations maintain compliance with policies set by regulatory
bodies, they need to know the following:

* **The requirements for what teams
must do, driven by legal requirements or external commitments (aka. privacy
policy and compliance controls). This includes translating the comprehensive
external legal requirements into requirements that are tailored to products and
services of the organizations. 
* **Where the user data is stored or processed.** This includes understanding the
different systems and databases that store or process user data, as well as the
physical locations where user data is stored.
* **Which privacy policy or compliance control applies to
the system that stores/processes user data (aka. data mapping).** This includes
understanding how the organization's privacy policies and controls are applied to
different systems and databases. 
* **The visibility of the privacy compliance.** This includes
being able to track and monitor the organization's compliance with its privacy
policies and applicable laws and regulations.

PMAP provides a solution for the first three problems. We are working on a
solution to provide visibility of privacy compliance in the near future.
```

Reply:
btw, this part doesn't seem properly formatted. You need an empty line before bullet points.